.github/workflows/windows.yml

```diff
@@ -138,7 +137,6 @@ jobs:
         with:
           set-safe-directory: true
           repository: open-quantum-safe/liboqs
-          ref: 0.10.1
```

Comment:
Suggest replacing with main (easier to find/correctly activate again if we need to fix to a specific `liboqs` release again)